app/helpers/base_presenter_helper.rb

```diff
@@ -0,0 +1,5 @@
+module BasePresenterHelper
+  def urlify(uri)
+    URI.parse(uri).scheme ? uri : "http://#{uri}"
```

Comment:
what if domain works on SSL?

Reply:
It's ok, because usually sites have redirect to https